Lib/test/test_statistics.py

```diff
@@ -1,7 +1,5 @@
-x = """Test suite for statistics module, including helper NumericTestCase and
-approx_equal function.
-
-"""
+x = ("Test suite for statistics module, including helper NumericTestCase and "
+     "approx_equal function.")
```

Comment:
Where is it used?

Reply:
As per your comment, I reverted the unnecessary code.
Thank you for your review :)